sky/shell/ui/engine.cc

```diff
@@ -130,6 +130,7 @@ void Engine::SetServices(ServicesDataPtr services) {
       mojo::ConnectToService(services_->services_provided_by_embedder.get(),
                              &vsync_provider);
     }
+    animator_->Reset();
```

Comment:
LGTM, but you need to do this for `scene_scheduler` too.  That as something I just landed and will eventually replace VsyncProvider.